ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -321,7 +321,11 @@ class GraphsContainer extends React.Component {
               />
               <VictoryAxis
                 tickCount={10}
-                tickFormat={x => moment.utc(x).format('MMM DD')}
+                tickFormat={x =>
```

Comment:
The problem I see with this approach is that we're relying on a default date that the graphing library is providing when there isn't data. This default value could change at any time and we may not know about it. It's not actually a default date provided by moment - x is a Date object that is being formatted to a localized time by moment. I think a more stable approach is to determine where the source of data is coming from for each graph and to test for it - if it's empty, show the current day's date and if it isn't, show the date provided by x. 

Also, the same code is being written twice so this is something that you'd want to write once and store in a variable or function.

Reply:
I will put the code into one func, or var and use it in both places. Thank you for the feedback :)